app/views/users/show.html.erb

```diff
@@ -1 +1,19 @@
-<h1>Users Show </h1>
\ No newline at end of file
+<%= render 'shared/users', user: @user %>
+
+<div>
+
+    <div>
+        <h1>Bio</h1>
+        <p><%= @user.bio %></p>
+    </div>
+<% User.recent_posts(@user).each do |post| %>
+    <div>
+        <h1>post <%= post.title %></h1>
+        <p><%= post.text %></p>
+        <p>Comments: <%= post.comments_counter %> Likes: <%= post.likes_counter %></p>
+    </div>
+<%end%>
+    
+
+<a href="/users/<%= @user.id%>/posts" >See all Posts</a>
+</div>
```

Comment:
- [ ] kindly match the wireframe given 
your design :point_down: 
![Screenshot from 2023-07-20 22-23-31](https://github.com/joyapisi/blog_app/assets/66097371/45c7ac56-dcbe-4c6b-8fa0-01eb7d28965a)
wireframe :point_down: 
![Screenshot from 2023-07-20 22-23-13](https://github.com/joyapisi/blog_app/assets/66097371/090505f6-5f29-46dd-9f40-319ad5651f2c)